samples/layouts/dock-manager/add-content-runtime/index.html

```diff
@@ -0,0 +1,40 @@
+<html>
+  <head>
+    <title>DockManagerAddContentRuntime</title>
+    <meta charset="UTF-8" />
+
+    <link rel="shortcut icon" href="https://static.infragistics.com/xplatform/images/browsers/wc.png" >
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kanit&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+    <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v6.css" type="text/css" />
+  </head>
+
+  <body>
+    <div id="root">
+      <div class="container sample">
+          <div style="margin-bottom: 5px;">
+            <igc-button id="contentPaneBtn" variant="outlined">Add content pane</igc-button>
+            <igc-button id="tabGroupPaneBtn" variant="outlined">Add document</igc-button>
+            <igc-button id="floatingPaneBtn" variant="outlined">Add floating pane</igc-button>
+          </div>
```

Comment:
For a more consistent look with the Hiding Panes sample I would suggest this change:
```suggestion
          <div class="options horizontal">
            <button id="contentPaneBtn">Add content pane</button>
            <button id="tabGroupPaneBtn">Add document</button>
            <button id="floatingPaneBtn">Add floating pane</button>
          </div>
```